### DISCUSSION TITLE
pnpm CLI opens new window

### DISCUSSION THREAD

**Discussion by xxxace** [pnpm/pnpm#7971] [Q&A]
When I run pnpm install or any CLI command, a new window pops up and closes automatically after the command finishes executing.

![image](https://github.com/pnpm/pnpm/assets/60084325/cef4992f-d8ce-4431-95a3-f27dfa97b06c)

any way to bring it back to the terminal? thanks！！！！

**Comment by xxxace** [comment]
hello???

**Reply by learyt** [reply]
Incredibly late, but this issue is suddenly happening to me too. This seems related to the issue https://github.com/pnpm/pnpm/issues/5834
